changelog.d/15134.feature

```diff
@@ -0,0 +1 @@
+Allow use of the `/filter` Client-Server APIs on workers.
```

Comment:
I think we also need to update:

* `docker/configure_workers_and_start.py` so complement uses this
* https://github.com/matrix-org/sytest/blob/d1c931af1179848526d323f3955a9f21f81d600f/lib/SyTest/Homeserver/Synapse.pm#L1257 so sytest uses this.

Reply:
And yes this is annoying, see #12139.

Reply:
→ https://github.com/matrix-org/sytest/pull/1331